tests/tools.py

```diff
@@ -119,45 +119,8 @@ def running_under_travis():
     return 'TRAVIS' in os.environ and os.environ['TRAVIS'] == 'true'
 
 
-def get_osutil_for_travis():
-    distro_name = os.environ['_system_name'].lower()
-    distro_version = os.environ['_system_version']
-
-    if distro_name == "ubuntu" and distro_version == "14.04":
-        return Ubuntu14OSUtil()
-
-    if distro_name == "ubuntu" and distro_version == "16.04":
-        return Ubuntu16OSUtil()
-
-
-def mock_get_osutil(*args):
```

Comment:
The whole method is obsolete since there is no need to do custom checks in Travis anymore.